From 31bbc46f09b56d8d36a1a11380742db16433ead3 Mon Sep 17 00:00:00 2001
From: Marco Dalla Libera <marco.dallas95@gmail.com>
Date: Wed, 29 May 2013 20:42:21 +0200
Subject: [PATCH 1/6] Maybe, in favor of users, you want to add my software to
 your readme

My java programs facilitate the installation and usage of your module, you can found them here:
http://marcodallas.github.io/acpi_call_GUI/
https://github.com/marcoDallas/acpi_call_GUI_Fedora

Best regards from
Marco
---
 README.md | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/README.md b/README.md
index cf7541a..86037ef 100644
--- a/README.md
+++ b/README.md
@@ -5,6 +5,8 @@ This module is to be considered a proof-of-concept and has been superseeded by
 projects like [bbswitch](https://github.com/Bumblebee-Project/bbswitch). It
 allows you to tamper with your system and should be used with caution.
 
+Want to use acpi_call via graphic interface? [Click here](https://github.com/mkottman/acpi_call/edit/master/README.md#graphic-interface)
+
 Usage:
 
     echo '<call>' | sudo tee /proc/acpi/call
@@ -49,6 +51,15 @@ The status after a call can be read back from `/proc/acpi/call`:
    above types (integer, string and buffer) and other package types
 
 
+#### Graphic interface
+
+Found this too difficult? Try to use these programs provided by Marco Dalla Libera and do it via graphic interface:
+
+* [acpi_call_GUI (Ubuntu and other debian-based distributions)](http://marcodallas.github.io/acpi_call_GUI/)
+* [acpi_call_GUI_Fedora (Fedora version)](https://github.com/marcoDallas/acpi_call_GUI_Fedora)
+
+***
+
 Copyright (c) 2010: Michal Kottman
 
 This program is free software: you can redistribute it and/or modify
-- 
1.8.2.1

